docs/source/howto/run_codes.rst

```diff
@@ -144,6 +144,38 @@ After the setup and configuration have been completed, let's check that everythi
 
 This command will perform various tests to make sure that AiiDA can connect to the computer, create new files in the scratch directory, retrieve files and query the job scheduler.
 
+.. _how-to:run-codes:computer:performance:
+
+Computer performance configuration
+----------------------------------
+
+Some compute resources, particularly large supercomputing centres, may not tolerate submitting too many jobs at once, executing scheduler commands too frequently or opening too many SSH connections.
+
+  * Limit the number of jobs in the queue.
+
+    Set yourself a limit for the maximum number of workflows to submit, and submit new ones only once previous workflows start to complete (in the future `this might be dealt with by AiiDA automatically <https://github.com/aiidateam/aiida-core/issues/88>`_).
```

Comment:
```suggestion
    Set yourself a limit for the maximum number of workflows to submit, and submit new ones only once previous workflows start to complete.
```
I generally think that plans for implementations should be documented as part of the roadmap and not as part of the documentation. Especially in the how-tos, users should not need to care about development plans IMO.